src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -12,6 +13,14 @@ public class DriverServiceImpl implements DriverService {
     @Inject
     private DriverDao driverDao;
 
+    @Override
+    public Driver findByLogin(String login) throws AuthenticationException {
+        return driverDao.findByLogin(login).orElseThrow(() ->
+                new AuthenticationException("The user with login "
+                        + login + " doesn't exist!")
+        );
+    }
```

Comment:
Please use exception class that extends RuntimeException and not tightly connected to authentication.
There may be a lot of scenarios when we try to find user that are not connected to authentication process at all
```suggestion
    public Driver findByLogin(String login) {
        return driverDao.findByLogin(login).orElseThrow(() ->
                new NoSuchElementException("The user with login "
                        + login + " doesn't exist!")
        );
    }
```